src/Endpoints/Recipient.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace MailerSend\Endpoints;
+
+use Assert\Assertion;
+use MailerSend\Helpers\GeneralHelpers;
+
+class Recipient extends AbstractEndpoint
+{
+    protected string $endpoint = 'recipients';
+
+    public const DEFAULT_LIMIT = 25;
+    public const MAX_LIMIT = 100;
+    public const MIN_LIMIT = 10;
+
+
+    /**
+     * @param int|null $limit
+     * @param int|null $page
+     * @return array
+     * @throws \JsonException
+     * @throws \MailerSend\Exceptions\MailerSendAssertException
+     * @throws \Psr\Http\Client\ClientExceptionInterface
+     */
+    public function get(?int $limit = self::DEFAULT_LIMIT, ?int $page = null): array
+    {
+        if ($limit) {
+            GeneralHelpers::assert(
+                fn () => Assertion::min($limit, self::MIN_LIMIT, 'Minimum limit is ' . self::MIN_LIMIT . '.') &&
+                    Assertion::max($limit, self::MAX_LIMIT, 'Maximum limit is ' . self::MAX_LIMIT . '.')
+            );
+        }
+
+        return $this->httpLayer->get(
+            $this->buildUri($this->endpoint),
+            array_filter([
+                'limit' => $limit,
+                'page' => $page
+            ])
+        );
+    }
```

Comment:
I'm missing `domain_id` parameter here

Reply:
https://developers.mailersend.com/api/v1/recipients.html#get-recipients

Reply:
I have checked with the current app code. We have domain filter there. Maybe we should update our docs?

Reply:
yeah, its missing in the docs too, most likely will be missing in all SDKs :)

Reply:
whoops :)

Reply:
@doobas I double checked this one, I will add `domain_id` optional param, and update out developers doc aswell.

Reply:
@AbdelilahLbardi i did update dev docs before